### ISSUE TITLE
Blank Window

### ISSUE THREAD

**Issue by Madagarian** [aleganza/Akuse#3]
Just installed the current beta release, signed into Anilist and I'm met with a blank gray/black screen. Restarting shows a vertical white box that states it's logged into Anilist, then it changes to a full-screen white window before changing to gray/black. Neither click nor any keyboard inputs seem to do anything.

Edition	Windows 11 Home
Version	22H2
Installed on	‎10/‎22/‎2022
OS build	22621.2283
Experience	Windows Feature Experience Pack 1000.22662.1000.0
Device Surface Pro 7

**Comment by aleganza** [comment]
Hello, after a few seconds you see the grey window, the app should load without problem (Tested on Windows 11). Since the loading isn't much efficent, your problem may be related to your connession or to your anilist profile page (maybe if you have many anime in your lists). I would suggest you to try again to open the app, and when you see the grey screen just wait for its loading. If in a bunch of time (a minute or two if necessary) it still doesn't load, the problem is unlikely related to you, so message me here again so we can look for another solution.

**Comment by Madagarian** [comment]
Hi, thanks for your reply. Idiscovered the cause. I had created a new AL
account and had no items in any lists. Your program doesn't load anything
at all if there is no lists. Once I added a show to my account, I
relaunched and it loaded the entire top bar, my continue watching, and the
rest of the headers and their content. I presume you just need to make sure
the ui loads properly for new/blank accounts.

Unrelated, don't want to make a mew issue for something so minor, progress
and score are manditory in thlist editor; I can't add a show to anything
(planning or otherwise) without scoring something I'm not watching/haven't
finished. And sam, for progress, eithet remove mandatory or add a default 0.
